app/build.gradle.kts

```diff
@@ -60,3 +61,21 @@ dependencies {
     implementation(libs.hilt.android)
     kapt(libs.hilt.compiler)
 }
+
+sonar {
+    properties {
+        property("sonar.projectKey", "bitwarden_passwordless-android")
+        property("sonar.organization", "bitwarden")
+        property("sonar.host.url", "https://sonarcloud.io")
+        property("sonar.sources", "app/src/main/,passwordless/src/main/")
+        property("sonar.tests", "app/src/test/,passwordless/src/test/")
```

Comment:
🎨 The "passwordless" module wouldn't need to be here since it's listed separately below.